docs/operations/metrics.md

```diff
@@ -180,6 +180,7 @@ These metrics are only available if the RealtimeMetricsMonitor is included in th
 |`ingest/handoff/count`|Number of handoffs that happened.|dataSource, taskId, taskType.|Varies. Generally greater than 0 once every segment granular period if cluster operating normally|
 |`ingest/sink/count`|Number of sinks not handoffed.|dataSource, taskId, taskType.|1~3|
 |`ingest/events/messageGap`|Time gap between the data time in event and current system time.|dataSource, taskId, taskType.|Greater than 0, depends on the time carried in event |
+|`ingest/notices/queueSize`|Number of pending notices to be processed by the coordinator|dataSource.|Greater than or equal to 0, should not be a very high number |
```

Comment:
I would expect a typical value to be 0 and occasionally in the low single-digits, anything greater than 0 indicates that there is some waiting time.

Reply:
will update it accordingly.